pulpcore/cli/ostree/repository.py

```diff
@@ -9,6 +8,8 @@
     PulpRemoteContext,
     PulpRepositoryContext,
 )
+from pulp_glue.common.i18n import get_translation
+from pulp_glue.core.context import PulpArtifactContext
```

Comment:
Ohhh, this could have changed from the first incarnation of pulp_glue...

:+1: